app/server-routes/src/main/scala/org/bitcoins/server/util/ServerBindings.scala

```diff
@@ -0,0 +1,27 @@
+package org.bitcoins.server.util
+
+import akka.http.scaladsl.Http
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{ExecutionContext, Future}
+
+case class ServerBindings(
+    httpServer: Http.ServerBinding,
+    webSocketServerOpt: Option[Http.ServerBinding]) {
+
+  private val terminateTimeout = 5.seconds
```

Comment:
would be nice if this was configurable

Reply:
Not going to do this for now because of this: https://github.com/bitcoin-s/bitcoin-s/issues/3934